### PR TITLE
Doc configuration update

### PR DESCRIPTION
**Pull request by aburrell**
Updates to address #48 including
- PEP8 updates to conf
- Switching from pytest-flake8 to flake8
- Fixing docstring style violations found by sphinx
- Cycled Python version testing support
- Updated README in preparation for the next release
- Fixed the zenodo author name style.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div align="left">
         <img height="0" width="0px">
-        <img width="20%" src="/docs/figures/pysatMadrigal.png" alt="pysatMadrigal" title="pysatMadrigal"</img>
+        <img width="20%" src="https://raw.githubusercontent.com/pysat/pysatSpaceWeather/main/docs/figures/pysatMadrigal.png" alt="pysatMadrigal" title="pysatMadrigal"</img>
```
Comment (aburrell) [inline README.md:3]:
```suggestion
        <img width="20%" src="https://raw.githubusercontent.com/pysat/pysatMadrigal/main/docs/figures/pysatMadrigal.png" alt="pysatMadrigal" title="pysatMadrigal"</img>
```

File: setup.cfg [hunk 2]
```diff
@@ -53,14 +52,12 @@ omit = */instruments/templates/
...
 markers =
     all_inst: tests all instruments
     download: tests for downloadable instruments
     no_download: tests for instruments without download support
     first: first tests to run
-    second: second tests to run
-flake8-ignore =
-  *.py W503
-  docs/conf.py ALL
+    second: second tests to run
```
Comment (aburrell) [inline setup.cfg:63]:
```suggestion
    second: second tests to run
    
```

File: .travis.yml [hunk 3]
```diff
@@ -2,13 +2,15 @@ language: python
 dist: xenial
 jobs:
   include:
-    - python: '3.6'
-      script: pytest --cov=pysatMadrigal/
     - name: '3.7 with flake8'
       python: '3.7'
-      script: pytest --flake8 --cov=pysatMadrigal/
+      script:
+          - flake8
```
Comment (jklenzing) [inline .travis.yml:8]:
```suggestion
          - flake8 . --count --select=E,F,W
```
Flake8 will ignore warnings unless we tell it to error on these as well  (pytest-flake8 errors on warnings by default).

**Review by jklenzing — APPROVED** [review]
minor update to keep the flake8 linting at the same level

**Pull request merged**